test/workspaces/setup_py-app/setup.py

```diff
@@ -6,7 +6,6 @@
     name="test_package",
     version="1.0.2",
     install_requires=[
-        "Jinja2==2.7.2",
```

Comment:
This workspace wasn't used in tests. I removed the Jinja2 for installation compatibility with python versions in tests.